modules/docs/mdx/MAINTAINING.mdx

```diff
@@ -380,3 +382,79 @@ describe('Example Codemod', () => {
 ```
 
 And that's it! You're done. Stage your changes, commit, and push up a PR.
+
+## Deprecations
+
+We add the [@deprecated](https://jsdoc.app/tags-deprecated.html) JSDoc tag to code we plan to remove
+in a future major release. This signals consumers to migrate to a more stable alternative before the
+deprecated code is removed.
+
+Add the `@deprecated` tag to the JSDoc comment directly above all exported declarations you wish to
+deprecate. Create a new JSDoc comment if one doesn't already exist.
+
+```tsx
+/**
+ * ...existing JSDoc comment, if present...
+ *
+ * @deprecated ⚠️ ${Deprecated Name} has been deprecated and will be removed in a future major release. ${Provide a migration strategy}
+ */
+export...
+```
+
+The provided migration strategy can take on any form as long as it gives the consumer a path forward
+once the deprecated code is removed.
+
+For example, we'll deprecate a component in our Main package before replacing it with an updated
+version of the component in our Preview or Labs packages. Note the optional Storybook link to the
+updated component.
+
+```tsx
+/**
+ * @deprecated ⚠️ Status Indicator in Main has been deprecated and will be removed in a future major release. Please use [`Status Indicator`](https://workday.github.io/canvas-kit/?path=/docs/preview-status-indicator--basic) in Preview instead.
```

Comment:
Moving "in Preview" to the link and removing the backticks since "Status Indicator in Preview" is not a component name.
```suggestion
 * @deprecated ⚠️ Status Indicator in Main has been deprecated and will be removed in a future major release. Please use [Status Indicator in Preview](https://workday.github.io/canvas-kit/?path=/docs/preview-status-indicator--basic) instead.
```

Reply:
ah ok